packages/react-core/src/components/Button/Button.tsx

```diff
@@ -58,6 +64,8 @@ export interface ButtonProps extends Omit<React.HTMLProps<HTMLButtonElement>, 'r
   inoperableEvents?: string[];
   /** Adds inline styling to a link button */
   isInline?: boolean;
+  /** Adds small or large styling to the button */
+  size?: 'sm' | 'default' | 'lg';
```

Comment:
Agreeing with @mcoker's feedback here, looks like we want to reorder the sizes within the prop too

```suggestion
  size?: 'default' | 'sm' | 'lg';
```